### PR TITLE
Update demos.js

### PR DESCRIPTION
**Pull request by AlleyWei**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by AlleyWei** [comment]
cc: @fengypants

**Comment by fengypants** [comment]
Should we put the TechDocs video after "Intro to Backstage" instead of at the bottom of the page?

**Comment by AlleyWei** [comment]
Yeah, I can make that adjustment and move it up @fengypants

File: microsite/pages/en/demos.js [hunk 1]
```diff
@@ -78,6 +78,25 @@ const Background = props => {
...
 
+<Block className="stripe-bottom bg-black-grey">
+        <Block.Container style={{ justifyContent: 'flex-start' }}>
+          <Block.TextBox>
+            <Block.Title>Make documentation easy</Block.Title>
+            <Block.Paragraph>
+              Documentation! Everyone needs it, no one wants to create it, and no one can ever find it. Backstage follows a “docs like code” approach: you write documentation in Markdown files right alongside your code. This makes documentation easier to create, maintain, find — and, you know, actually use. This demo video showcases Spotify’s internal version of TechDocs. Learn more about the [open version of TechDocs]. (https://backstage.io/blog/2020/09/08/announcing-tech-docs)
+            </Block.Paragraph>
+            <Block.LinkButton
+              href={'https://youtu.be/mOLCgdPw1iA '}
```
Comment (fengypants) [inline microsite/pages/en/demos.js:89]: Looks like there's an extra space at the end of the link.

Comment (AlleyWei) [reply microsite/pages/en/demos.js]: Adjusted, thanks.

**Comment by stefanalund** [comment]
Replaced by https://github.com/spotify/backstage/pull/2335

**Pull request closed without merging**